homeassistant/components/recorder/db_schema.py

```diff
@@ -167,18 +171,27 @@ def __repr__(self) -> str:
         return (
             "<recorder.Events("
             f"id={self.event_id}, type='{self.event_type}', "
-            f"origin_idx='{self.origin_idx}', time_fired='{self.time_fired}'"
+            f"origin_idx='{self.origin_idx}', time_fired='{self.time_fired_isotime}'"
             f", data_id={self.data_id})>"
         )
 
+    @property
```

Comment:
Not sure if this field is requested a lot, but we could consider using [`@cached_property`](https://docs.python.org/3/library/functools.html#functools.cached_property)

Reply:
It's currently only called in __repr__. I only broke it out to improve the code readability so I messed it up on the first pass.